Automattic/wp-calypso
# Magic Login: Use Jetpack Colours

#### Changes proposed in this Pull Request

* During the Jetpack Connect flow, clicking "Email me a login link" results in the colours for the flow being dropped. This PR proposes changing that.

#### Testing instructions

1. Open an incognito tab and try to link a Jetpack site to WordPress.com
2. When asked to login, click "Email me a login link"
3. Verify you're redirected to `log-in/jetpack/link`

<img width="903" alt="Screenshot 2019-07-28 at 15 21 39" src="https://user-images.githubusercontent.com/43215253/62007929-6d8afe80-b14b-11e9-8ae5-a5df5801a552.png">

<img width="811" alt="Screenshot 2019-07-28 at 15 22 47" src="https://user-images.githubusercontent.com/43215253/62007944-98755280-b14b-11e9-8bfc-6c5d23c7a6fe.png">

Verify that "Enter a password" and "Back" redirects back to the Jetpack login page. Then visit `/log-in` and verify that nothing there has changed including the links at the bottom, and that `/log-in/link` doesn't have the Jetpack logo or colours.

This does not change the behaviour from after the emailed link is clicked, including the colours after the link is expired. I think that it's not possible in a PR to add `jetpack` to the link in the email sent. Also, signing in doesn't redirect back to the Jetpack login page (it redirects to the Reader), but this PR only intends to change the colours.

cc @simison 

Fixes #33781
